main.js

```diff
@@ -0,0 +1,25 @@
+// document.querySelector('.menu-bars').addEventListener('click', () => {
+//   document.querySelector('#mobile-nav').classList.toggle('invisible');
+// });
+
+// document.querySelector('#close').addEventListener('click', () => {
+//   document.querySelector('#mobile-nav').classList.toggle('invisible');
+// });
+
+const logic = () => {
+  document.querySelector('.menu-bars').addEventListener('click', () => {
+    document.querySelector('#mobile-nav').classList.toggle('invisible');
+  });
+
+  document.querySelector('#close').addEventListener('click', () => {
+    document.querySelector('#mobile-nav').classList.toggle('invisible');
+  });
+
+  document.querySelectorAll('.mobile-link').forEach((link) => {
+    link.addEventListener('click', () => {
+      document.querySelector('#mobile-nav').classList.toggle('invisible');
```

Comment:
- [ ] Please follow the `DRY` rule, your previous code reviewer asked you to change it but as I can see you are still repeating your statement in lines `11`, `15` and `20`.  As of now it seems like there is no issue to repeat the code 2-3 times but consider the `scenario` where your project is `scaling` and you have to add more and more logical conditions then it will be `difficult` for any person to understand your code and `maintain` it. 

- Kindly consider the below example and can create a `variable` for all the above-mentioned repeat lines of code. 
  > const mobileNav = document.querySelector('#mobile-nav');